cli/spark/endpoints/init.py

```diff
@@ -8,14 +8,29 @@
 def setup_parser(parser: argparse.ArgumentParser):
     parser.add_argument('--global', dest='global_flag', action='store_true',
                         help="Create a .aztk/ folder in your home directory for global configurations.")
+    software_parser = parser.add_mutually_exclusive_group()
+    software_parser.add_argument('--python', action="store_true", required=False)
+    software_parser.add_argument('--r-base', action="store_true", required=False)
+    software_parser.add_argument('--java', action="store_true", required=False)
+    software_parser.add_argument('--scala', action="store_true", required=False)
```

Comment:
Will it be confusing to users to split out java & scala? They're essentially the same thing and may leave customers wondering why we have split them out. Thoughts?

Reply:
No preference, just not sure how to word the flag if we combine them.

Under the hood, `--java`, `--scala` and no flag all do the same thing.

Reply:
yeah.. I just can't think of a word that combines them both, like .NET but for java... I was thinking 'default' but that doesn't seem right either.

Reply:
maybe just 'base'? and add some verbosity to the CLI and docs that this will use the out-of-the-box java/scale environment?